### PR TITLE
chore: document disabling affinity assistant for scan v2

### PR DESCRIPTION
**Pull request by garethjevans**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: scst-scan/install-app-scanning.hbs.md [hunk 1]
```diff
@@ -28,6 +28,14 @@ When you install SCST - Scan 2.0, you can configure the following optional prope
 
 >**Note** If the StorageClass you select does not have a node limit but uses the node storage, such as hostpath, the nodes must have large enough disks. For example, if a scan creates a 2Gi volume on a hostpath type storage class, `2Gi * number of AMR images` indicates how much storage this cluster needs overall. `2Gi * number of AMR images / number of nodes` indicates how much storage each node needs.
 
+It is also strongly recommended that you disable the Tekton Affinity Assistant to reduce the number of pod scheduling issues when using Scan v2.  This can be acheived by adding the following to your `tap-values.yaml`:
```
Comment (mdc81) [inline scst-scan/install-app-scanning.hbs.md:31]: Going by our findings, shouldn't it be must, not a recommendation?

**Comment by YugangW** [comment]
Hi @garethjevans @mdc81 Please let us know when this PR is ready to be merged and kindly advise if this change applies to other TAP versions older than v1.8. Thanks.

**Comment by mdc81** [comment]
> Hi @garethjevans @mdc81 Please let us know when this PR is ready to be merged and kindly advise if this change applies to other TAP versions older than v1.8. Thanks.

@YugangW This will be applicable for 1.7.1 onwards for the clusters which have PSA PSS enabled, for example TKGs TKr 1.26. @garethjevans Please correct if my statement is not correct..

**Comment by garethjevans** [comment]
@mdc81 I have to disable affinity assistant on GKE to get more than a small number of concurrent scans to pass.

**Comment by mdc81** [comment]
thanks for clarifying @garethjevans. Then then original wordings sound good.

**Pull request merged**